Automattic/woocommerce-payments
# Card testing: replace enable toggle with account eligibility flag

Fixes #4178 

#### Changes proposed in this Pull Request
Replace toggle option for card testing with eligibility flag implemented in 1915-gh-Automattic/woocommerce-payments-server.








#### Testing instructions

* fingerprint meta should always be sent on payment intent
* set card testing eligibility flag to `true` and ensure `wcpay-fraud-prevention-token` hidden field is available in checkout form

-------------------


**Post merge**
